deutsches-archaologisches-institut.csl

```diff
@@ -23,17 +28,17 @@
         <group delimiter=". ">
           <choose>
             <if variable="author">
-              <names variable="editor">
+              <names variable="editor" font-variant="normal">
```

Comment:
You should be able to delete the ` font-variant="normal"` here (which is the default).